platform/ios/CHANGELOG.md

```diff
@@ -23,12 +25,14 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 
 ### Annotations and user interaction
 
+* Fixed several bugs and performance issues related to the use of annotations backed by `MGLAnnotationImage`s. These fixes also apply to images used to represent icons in `MGLSymbolStyleLayer`s. ([#9213](https://github.com/mapbox/mapbox-gl-native/pull/9213))
```

Comment:
Specifically, this fixed #3185, which was previously a major issue for iOS developers. Maybe we can say that it eliminated the too-many-sprites problem for all practical purposes.